blockmanager.go

```diff
@@ -134,13 +133,33 @@ type headerNode struct {
 	hash   *chainhash.Hash
 }
 
+// The PeerNotifier exposes methods to notify peers of status changes to
```

Comment:
All comments should start with the name of the thing they are commenting per [Effective Go](https://golang.org/doc/effective_go.html) which this project follows.  In other words `PeerNotifier ...`.